.aspect/workflows/README.md

```diff
@@ -1,20 +1,15 @@
 # Aspect Workflows demonstration deployment
 
-This deployment of [Aspect Workflows](https://www.aspect.build/workflows) is configured to run on AWS + GitHub Actions.
+This deployment of [Aspect Workflows](https://www.aspect.build/workflows) is configured to run on GitHub Actions.
```

Comment:
Don't want to emphasize the cloud?

Reply:
Not anymore since I've taken the terraform out. Its transparent as far as the user repository configuration is concerned.